troubleshoot/ingest/fleet/common-problems.md

```diff
@@ -362,7 +362,7 @@ The ZIP archive containing diagnostics information will include the raw events o
 
 **Get the diagnostics bundle using the CLI**
 
-Run the following command to generate a zip archive containing diagnostics information that the Elastic team can use for debugging cases.
+Run the following [Agent command](https://www.elastic.co/docs/reference/fleet/agent-command-reference#elastic-agent-diagnostics-command) to generate a zip archive containing diagnostics information that the Elastic team can use for debugging cases.
```

Comment:
Thank you! A few thoughts:

* Even though we're already in the Elastic Agent docs, we should use the full name, "Elastic Agent" to distinguish between Elastic Agent and APM Agents.
* Because the page you're linking to is in the same repo, you should use one of our two [internal link syntaxes](https://docs-v3-preview.elastic.dev/elastic/docs-builder/tree/main/syntax/links#internal-links). 
* I would change the link to be the name of the command to give more context as to where the user is being linked off to.

```suggestion
Run the [`diagnostics` command](/reference/fleet/agent-command-reference#elastic-agent-diagnostics-command) to generate a zip archive containing diagnostics information that the Elastic team can use for debugging cases:
```

Reply:
Clarified in https://github.com/elastic/docs-builder/pull/1447